10_files_lab/fileslab.txt

```diff
@@ -8,30 +8,69 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
   - .highscore
 
 Write here the commands you used:
+mkdir lab
+cd lab
+touch a.out
+touch hero.c
+touch enemy.c
+touch game.c
+touch main.c
+touch human.h
+touch monster.h
+touch highscore.h
 
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
 Write here the commands you used:
+cd lab
+mkdir Music
+mkdir Misc
```

Comment:
same as with ```touch``` mkdir can accept multiple parameters.
but it's totally fine to also do it this way, it's a matter of style :)